include/secp256k1.h

```diff
@@ -452,7 +452,13 @@ SECP256K1_API int secp256k1_ecdsa_signature_serialize_compact(
  *           0: incorrect or unparseable signature
  *  Args:    ctx:       a secp256k1 context object, initialized for verification.
  *  In:      sig:       the signature being verified (cannot be NULL)
- *           msg32:     the 32-byte message hash being verified (cannot be NULL)
+ *           msghash32: the 32-byte message hash being verified. The verifier
+ *                      must make sure to apply a cryptographic hash function to
+ *                      the message by itself and not accept an msghash32 value
+ *                      directly. Otherwise, it would be easy to create a
+ *                      "valid" signature without knowledge of the secret key.
+ *                      See also https://bitcoin.stackexchange.com/a/81116/35586
+ *                      for more background on this topic. (cannot be NULL)
```

Comment:
maybe move the "cannot be NULL" up again.

Otherwise it's "See also https://bitcoin.stackexchange.com/a/81116/35586 for more background on this topic. (cannot be NULL)", which confused me. We could also move the explanation to the body alternatively.

Reply:
Fixed